test/integration/integration_test.go

```diff
@@ -102,6 +113,25 @@ func (i *integrationTest) setup() {
 		}
 		i.setupComplete = true
 	}
+
+	from := flag.CommandLine
+	if vflag := from.Lookup("v"); vflag != nil {
+		fmt.Fprintf(os.Stdout, "integration_test.go: found -v flag from starter bash script with initial glog level %d\n", getLogLevel())
```

Comment:
it was just a function on not getting glog to work initially when I was developing ... I can switch it.